docs/changes/visualization/13810.api.rst

```diff
@@ -0,0 +1,2 @@
+Rename number-of-samples keyword ``nsamples`` to the module-standard
+``n_samples`` using the ``@deprecated_renamed_argument`` decorator.
```

Comment:
The fact that the keyword change was made using `@deprecated_renamed_argument` is an implementation detail that the typical user shouldn't have to care about. What the users are much more likely to be interested in is why the keyword was changed.

Reply:
Added more user-focused verbiage to change-log.